index.html

```diff
@@ -9,261 +9,393 @@
+             
+         </section>
+         <footer>
+            <div class="footer-container">
+                <div class="header-img-left">
+                    <img src="images/image-geometry_6.png" alt="image">
                 </div>
-                <div class="project-view">
-                        <span class="project-view-union">See this project</span>
+                <div class="contact">
+                    <div class="contact-description">
+                        <h3>Get started</h3>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+                    </div>
+                    <form action="https://formspree.io/f/xbjwgkqr" method="post">
+                        <input type="text" name="name" class="name" placeholder="Your name" maxlength="30" required>
+                        <input type="email" name="email" class="email" placeholder="Your email" required>
                         
-                        <span><img src="images/Union.png" alt="union"></span>
-                </div>
-              </div>
-            </div>
-        </div>
-    </section>
-    <section class="about">
-        <div class="about-me">
-            <h3>About</h3>
-            <P>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</P>
-    
-            <button name="view-project" type="button" id="about-view">
-                <span class="project-view-btn">Get my CV</span>
-                
-                <span><img id="cv-btn" src="images/ic_download.png" alt="union"></span>
-            </button>
-        </div>
-        <section class="langs-section">
-            <div class="langs">
-               <div class="content-list">
-                <h3>Languages</h3>
-                <ul>
-                    <li>JavaScript (ES5/ES6)</li>
-                    <li>Ruby</li>
-                    <li>HTML</li>
-                    <li>CSS</li>
-                </ul>
-               </div>
-              <div class="langs-img">
-                <img class="images" src="images/hash.png" alt="#">
-              </div>
-            </div>
-            <div class="frameworks">
-                <div class="content-list">
-                    <h3>Frameworks</h3>
-                <ul>
-                    <li>Bootstrap</li>
-                    <li>Ruby on rails</li>
-                    <li>RSpec</li>
-                    <li>Capibara</li>
-                    <li>Selenium</li>
-                </ul>
-    
+                        <textarea name="message" class="msg" cols="30" rows="10" placeholder="Enter your message here..." maxlength="500" required></textarea>
+                        <button class="btn-collabo-desk" type="submit" name="submit">Start collaboration</button>
+        
```

Comment:
- [ ] Kindly fix the button design to be like the Figma design in the mob version.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/155574943-6832219e-6ec1-4933-9764-f5cafdb8fd6a.png) | ![image](https://user-images.githubusercontent.com/35535130/155574887-ba37dacc-a0ec-4482-b1c6-b5739dc4ce36.png) |